File: JavaScript/7js 2/mar.js
/*
|--------------------------------------------------------------------------
| Debug the functions EASY
|--------------------------------------------------------------------------
|
*/

function multi(arr) {
   return arr.reduce((a, b) => a * b)
}


function add(arr) {
   return arr.reduce((a, b) => a + b)
}


function reverse(str) {
   return str.split('').reverse().join('');
}


console.log(multi([5, 1, 5]), 25);
console.log(add([9, 8, 5]), 22);
console.log(reverse("Hello World"), "dlroW olleH");

File: Python/7py 2/mar.py
"""
|--------------------------------------------------------------------------
| Debug the functions EASY
|--------------------------------------------------------------------------
|
"""

from math import prod
def multi(arr):
   return prod(arr)


def add(arr):
   return sum(arr)


def reverse(string):
   return string[::-1]


print(multi([8,2,5]),80)
print(add([1,15,3]),19)
print(add([7,8,6,5,4,9]),39)
print(reverse("Hello World"),"dlroW olleH")